### ISSUE TITLE
All IOS Card Transactions Failing

### ISSUE THREAD

**Issue by tinymobile** [square/in-app-payments-react-native-plugin#221]
### Describe the issue

Hi, we are seeing an urgent issue that has just started today (Saturday 30th) where all card transactions on our IOS app are suddenly failing.

We are using React Native square and the startCardEntryFlowWithBuyerVerification is immediately returning an SCA fail whatever we put through it.

It's failing on our store version of the app that's using version 1.7.2 of the react native plug in and it's also failing on our dev version that's using the latest 1.7.3 version of the react native plug in.

It's failing on devices running all different IOS versions (e.g. IOS 15, IOS 16 and IOS 17)

It's also failing all card types (e.g. VISA, AMEX).

It's still working fine on our Android version of the app.

It was fine until yesterday on IOS and we have not done an update to our app for a while, so nothing has changed our end.

Is there a problem at Square's end?

### To Reproduce

Fire startCardEntryFlowWithBuyerVerification from our app.

Put in any card details to the square card entry dialog.

It immediately fails SCA on any card we try.

### Expected behavior

Card is processed.

### Environment (please complete the following information):

IOS all versions.

**Comment by kirstiedownie** [comment]
Hi @tinymobile. Are you still seeing an issue?

**Comment by kuldip-simform** [comment]
Yes, I am facing this issue. I am testing in sandbox mode, and my payment is failing for every device, be it iOS or Android.
